### DISCUSSION TITLE
Figma: UI Kit and Templates

### DISCUSSION THREAD

**Discussion by arnodt-tech** [primefaces/community#2620] [Figma UI Kit]
We purchased UI Kit and Verona template for PrimeReact.  In the Figma states that we would be able to download the figma file for the template.  Can you please advise were can we download it as it is not in the download page?
![Capture](https://github.com/user-attachments/assets/fc2a541c-d30f-4212-b4c8-375823450fa9)

**Comment by onursenture** [comment]
Hi, For the Verona's Figma please check PrimeStore / My Templates / Download Files / Figma Design:

<img width="621" alt="image" src="https://github.com/user-attachments/assets/d48053b5-22ed-415a-af83-b3efa0754a5c">
